src/main/java/com/czertainly/core/dao/entity/acme/AcmeOrder.java

```diff
@@ -71,7 +71,7 @@ public Order mapToDto(){
         order.setAuthorizations(authorizations.stream().map(AcmeAuthorization::getUrl).collect(Collectors.toList()));
         order.setFinalize(getFinalizeUrl());
         if(certificateId != null){
-            order.setCertificate(getCertificateUrl());
+            order.setCertificate(getCertificateUrl()); // should we include URL of certificate if order is still in PROCESSING state?
```

Comment:
The certificate is optional according to the https://datatracker.ietf.org/doc/html/rfc8555#section-7.1.3 and should be included once the certificate has been issued. From my perspective, it does not make sense to have certificate URL in other state than VALID. Once VALID, it should have the URL where to download the certificate.

Reply:
I agree, I will update PR accordingly